heka/meta/heka.yml

```diff
@@ -0,0 +1,85 @@
+log_collector:
```

Comment:
You created an `heka/meta/heka.yml` while `heka` is still disabled in the `support` reclass. Do you mean to enable the heka support in `metadata/service/support.yml`?